src/solvers/imex_ark_tableaus.jl

```diff
@@ -0,0 +1,208 @@
+export ARS111, ARS121, ARS122, ARS233, ARS232, ARS222, ARS343, ARS443
+export IMKG342a, IMKG343a, IMKG353a, IMKG354a
+
+using StaticArrays: @SArray
+
+# ARS tableaus
+# From Section 2 of "Implicit-explicit Runge-Kutta methods for time-dependent
+# partial differential equations" by Ascher et al.
+# Each tableau is named ARSsσp, where s is the number of implicit stages, σ is
+# the number of explicit stages, and p is the order of accuracy
+
+const ARS111 = make_IMEXARKAlgorithm(;
+    a_exp = @SArray([0 0; 1 0]),
+    a_imp = @SArray([0 0; 0 1]),
+) # This is equivalent to OrdinaryDiffEq.IMEXEuler
+
+const ARS121 = make_IMEXARKAlgorithm(;
+    a_exp = @SArray([0 0; 1 0]),
+    b_exp = @SArray([0, 1]),
+    a_imp = @SArray([0 0; 0 1]),
+)
+
+const ARS122 = make_IMEXARKAlgorithm(;
+    a_exp = @SArray([0 0; 1//2 0]),
+    b_exp = @SArray([0, 1]),
+    a_imp = @SArray([0 0; 0 1//2]),
+    b_imp = @SArray([0, 1]),
+)
+
+const ARS233 = let
+    γ = (3 + sqrt(3)) / 6
+
+    make_IMEXARKAlgorithm(;
+        a_exp = @SArray([
+            0     0      0;
+            γ     0      0;
+            (γ-1) (2-2γ) 0;
+        ]),
+        b_exp = @SArray([0, 1/2, 1/2]),
+        a_imp = @SArray([
+            0 0      0;
+            0 γ      0;
+            0 (1-2γ) γ;
+        ]),
+        b_imp = @SArray([0, 1/2, 1/2]),
+    )
+end
```

Comment:
Let's make these functions that accept `::Type{FT}`, since we want type-stable code.